src/cmds/configure.py

```diff
@@ -79,38 +27,58 @@ def configure(args, parser):
                 change_made = True
         except ValueError:
             print("Node introducer address must be in format [IP:Port]")
-    if args.set_fullnode_port:
-        config["full_node"]["port"] = int(args.set_fullnode_port)
-        config["full_node"]["introducer_peer"]["port"] = int(args.set_fullnode_port)
-        config["farmer"]["full_node_peer"]["port"] = int(args.set_fullnode_port)
-        config["timelord"]["full_node_peer"]["port"] = int(args.set_fullnode_port)
-        config["wallet"]["full_node_peer"]["port"] = int(args.set_fullnode_port)
-        config["wallet"]["introducer_peer"]["port"] = int(args.set_fullnode_port)
-        config["introducer"]["port"] = int(args.set_fullnode_port)
+    if set_fullnode_port:
+        config["full_node"]["port"] = int(set_fullnode_port)
+        config["full_node"]["introducer_peer"]["port"] = int(set_fullnode_port)
```

Comment:
Looks like `--set-node-introducer` and `--set-fullnode-port` collides a bit, should we raise an error when you try to use both at the same time?

Reply:
Yeah that makes sense